hilti/runtime/src/tests/util.cc

```diff
@@ -490,9 +490,9 @@ TEST_CASE("replace") {
 TEST_CASE("rtrim") {
     CHECK_EQ(rtrim("", ""), "");
     CHECK_EQ(rtrim("", "abc"), "");
-    CHECK_EQ(rtrim("4d3c2b1a", "abc"), "4d3c2b1");
-    CHECK_EQ(rtrim("4d3c2b1ba", "abc"), "4d3c2b1");
-    CHECK_EQ(rtrim("4d3c2b1cba", "abc"), "4d3c2b1");
+    CHECK_EQ(rtrim("4d3c2b1c", "abc"), "4d3c2b1");
+    CHECK_EQ(rtrim("4d3c2b1bc", "abc"), "4d3c2b1");
+    CHECK_EQ(rtrim("4d3c2b1abc", "abc"), "4d3c2b1");
```

Comment:
What's the reason for this change?

Reply:
This is a workaround for this diagnostic:

```.console
error: `ba` should be `by`, `be`
  --> hilti/runtime/src/tests/util.cc:494:27
    |
494 |     CHECK_EQ(rtrim("4d3c2b1ba", "abc"), "4d3c2b1");
    |                            ^^
    |
```